app/scenes/HomeScreen/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Button, Platform, View, ActivityIndicator } from 'react-native';
+import { PropTypes } from 'prop-types';
+import styled from 'styled-components/native';
+
+import AppContainer from '@atoms/Container';
+import ItunesSearch from '@organisms/ItunesSearch';
+
+const Container = styled(AppContainer)`
+  margin: 30px;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  max-width: 320px;
+  align-self: center;
+`;
+
+const CustomButtonParentView = styled(View)`
+  margin-top: 40px;
+  max-width: 80px;
+  align-self: center;
+`;
+const instructions = Platform.select({
+  ios: 'Press Cmd+R to reload,\nCmd+D or shake for dev menu.',
+  android:
+    'Double tap R on your keyboard to reload,\nShake or press menu button for dev menu.'
+});
+
+class HomeScreen extends React.Component {
+  componentDidMount() {
+    this.requestFetchTracks();
+  }
+
+  requestFetchTracks = () => {
+    console.log(this.props.fetchTracks);
```

Comment:
```console.log()```
is good for playing around, lets not get this inside `main` branch
not recommended to use in production anyway